che-core-git-impl-native/src/main/java/org/eclipse/che/git/impl/nativegit/commands/CheckoutCommand.java

```diff
@@ -43,21 +45,31 @@ public Void execute() throws GitException {
 
         reset();
         commandLine.add("checkout");
-
-        if (createNew) {
-            commandLine.add("-b");
-            commandLine.add(branchName);
-        } else if (branchName != null) {
-            commandLine.add(branchName);
+        
+        if (filePaths != null && !filePaths.isEmpty())
```

Comment:
please reformat in accordance with other code. 
1. Curly braces should not use separate line. 
2. Tab characters are forbidden. Use spaces instead.
3. Indentation 4 spaces